### ISSUE TITLE
(require 'xah-fly-keys) shouldn't enable xah-fly-keys

### ISSUE THREAD

**Issue by matta** [xahlee/xah-fly-keys#39]
I followed the manual install instructions and had this in my .emacs:

```
(require 'xah-fly-keys)
(xah-fly-keys-set-layout "qwerty")
(xah-fly-keys 1)
```

Then I ran `eval-last-sexp` on `(require 'xah-fly-keys)` and had a broken Emacs.  The mode seemed to be partially enabled:

1. I saw "flykeys" in the mode line
1. I did not see the leading C or I indicating modality
1. I was in fact in command mode, with Dvorak keys bound.  I use Qwerty.

The elisp coding conventions request that 

> Simply loading a package should not change Emacs's editing behavior

See https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html

I found it pretty confusing.  It'd be great if the package didn't do this.

**Comment by matta** [comment]
I see issue https://github.com/xahlee/xah-fly-keys/issues/18 now, which basically covers this same issue.

**Comment by duianto** [comment]
I also ran into this issue yesterday when I started trying xah-fly-keys.

While following the instructions on: http://ergoemacs.org/misc/ergoemacs_vi_mode.html
After evaluating the first command `(require 'xah-fly-keys)` in the MELPA Install section, then the keys changed and they didn't match the qwerty layout image in the "What Does Command Mode Do?" section.

I managed to move the cursor with the arrow keys and `C-x C-e` evaluate the other two commands:
```elisp
(xah-fly-keys-set-layout "qwerty")
(xah-fly-keys 1)
```
But the layout still didn't match the qwerty layout, now that I read this issue I realize that it probably setup the Dvorak layout.

After restarting Emacs, the Qwerty xah-fly-keys layout now works as expected.
